sound/soc/sof/mediatek/adsp_helper.h

```diff
@@ -0,0 +1,49 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+/*
+ * Copyright (c) 2021 MediaTek Corporation. All rights reserved.
+ */
+
+#ifndef __ADSP_HELPER_H__
+#define __ADSP_HELPER_H__
+
+/*
+ * Global important adsp data structure.
+ */
+#define DSP_MBOX_NUM	3
+
+struct adsp_chip_info {
```

Comment:
Same with the structs. E.g. there is "struct sof_intel_hda_dev" for Intel-hw specific structs.